src/app/zap-templates/zcl/custom-types.xml

```diff
@@ -42,4 +43,10 @@ limitations under the License.
      <item name="type" type="DEVICE_TYPE_ID"/>
      <item name="revision" type="INT16U"/>
    </struct>
+  <struct name="FabricDescriptor">
+     <item name="FabricId" type="FABRIC_ID"/>
+     <item name="VendorId" type="INT16U"/> // Change INT16U to new type VendorID #2395
+     <item name="Label" type="OCTET_STRING" length="32"/> // Change this to CHAR_STRING once it is supported #6112
```

Comment:
Existing clusters like Basic already use CHAR_STRING and it seems to work fine. Probably it could be changed to CHAR_STRING now without an issue.

Reply:
Will double check but Vivien did not believe it was fully supported yet.

Reply:
Ok so CHAR_STRING works if it is a parameter in a command or an attribute. However it still does not work when it is a member of a list. I have made the UpdateFabricLabel be a CHAR_STRING but kept Label in the FabricDescriptor as an OCTET_STRING until we resolve.